src/App.css

```diff
@@ -36,3 +36,13 @@
     transform: rotate(360deg);
   }
 }
+
+.images {
+  height: 100px;
+  width: 100px;
+  margin-bottom: 0px;
+}
+
+#date {
+margin-bottom: 80px;
+}
```

Comment:
If you'd like to get rid of the bullet points in you `<li>` elements you can by giving them the `list-style-type` css property and setting it to `none`.